### DISCUSSION TITLE
upgrade errors

### DISCUSSION THREAD

**Discussion by andim27** [nylo-core/nylo#102] [General]
I catch an error when update from nylo 5.15 to 5.19.It depends on package intl(Something wrong in DateFormat(Fix issue [#483](https://github.com/dart-lang/i18n/issues/483)) if you use localisation in the app) They use now version 19.0 but last Nylo use 18.1. Please in future release setup intl ver > 19.0. I would like to note that Upgrading to the next version is becoming more and more difficult.
Addition: It would also be necessary to correct the build for Android in accordance with
[(https://docs.flutter.dev/release/breaking-changes/flutter-gradle-plugin-apply)]

**Comment by agordn52** [comment]
Hi @andim27,

Thanks for reporting this.

At the moment we can't upgrade to intl 0.19.0 because Nylo uses the `flutter_localizations` package from the SDK.
If you view this [file](https://github.com/flutter/flutter/blob/master/packages/flutter_localizations/pubspec.yaml) be able to see in the pubspec.yaml for `flutter_localizations` it's currently using intl 0.18.1 

If they upgrade to 0.19 then we'll be able to update this in the nylo_support package.

The latest version https://pub.dev/packages/nylo_framework shouldn't give you any issues 👍
